### PR TITLE
Redirect to App custom url scheme with token data as query params

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-server/pull/16447

Updated:
- Self-Managed Admin Guide > Configure Mattermost > Configuration Settings > Site Configuration
   - Added new config setting: App Custom URL Schemes

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1552,6 +1552,15 @@ Set the link for the support website.
 | This feature's ``config.json`` setting is ``"ReportAProblemLink": "https://about.mattermost.com/default-report-a-problem/"`` with string input.                                            |
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+App Custom URL Schemes
+^^^^^^^^^^^^^^^^^^^^^^
+
+Define valid custom URL schemes for redirect links provided by customer-built mobile Mattermost apps. This ensures users are redirected to the customer-built mobile app and not Mattermost's mobile client. When configured, after OAuth or SAML user authentication is complete, custom URL schemes sent by mobile clients are validated to ensure they don't include default schemes such as ``http`` or ``https``. Mobile users are then redirected back to the mobile app using the custom scheme URL provided by the mobile client. We recommend that you update your mobile client values as well with valid custom URL schemes.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:1558]:
```suggestion
Define valid custom URL schemes for redirect links provided by custom-built mobile Mattermost apps. This ensures users are redirected to the custom-built mobile app and not Mattermost's mobile client. 

When configured, after OAuth or SAML user authentication is complete, custom URL schemes sent by mobile clients are validated to ensure they don't include default schemes such as ``http`` or ``https``. Mobile users are then redirected back to the mobile app using the custom scheme URL provided by the mobile client. We recommend that you update your mobile client values as well with valid custom URL schemes.
```

**Review by justinegeffen — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]
Thanks

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -1552,6 +1552,17 @@ Set the link for the support website.
...
 
+App Custom URL Schemes
+^^^^^^^^^^^^^^^^^^^^^^
+
+Define valid custom URL schemes for redirect links provided by custom-built mobile Mattermost apps. This ensures users are redirected to the custom-built mobile app and not Mattermost's mobile client. 
+
+When configured, after OAuth or SAML user authentication is complete, custom URL schemes sent by mobile clients are validated to ensure they don't include default schemes such as ``http`` or ``https``. Mobile users are then redirected back to the mobile app using the custom scheme URL provided by the mobile client. We recommend that you update your mobile client values as well with valid custom URL schemes.
+
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"NativeAppSettings.AppCustomURLSchemes"`` with string input.                                                                                   |
```
Comment (anurag6713) [inline source/administration/config-settings.rst:1563]: ~~string input~~ .. it is array of strings .. Example: `['custom-app://', 'some-app://']`

Comment (cwarnermm) [reply source/administration/config-settings.rst]: Thanks so much, @anurag6713!

**Review by anurag6713 — APPROVED** [review]
LGTM! 🙂

**Pull request merged**